packages/studio-be/src/core/bots/bot-service.ts

```diff
@@ -219,15 +219,17 @@ export class BotService {
     }
   }
 
-  async exportBot(botId: string): Promise<Buffer> {
+  async exportBot(botId: string, { cloud }: { cloud: boolean } = { cloud: false }): Promise<Buffer> {
     const replaceContent: ReplaceContent = {
       from: [new RegExp(`/bots/${botId}/`, 'g')],
       to: [BOT_ID_PLACEHOLDER]
     }
+    const excludes = ['libraries/node_modules/**/*']
+    if (!cloud) {
+      excludes.push('models/**/*')
```

Comment:
If cloud is set to true, we don't exclude the models as we want them in the archive.

Reply:
:+1: